docs/guides/use_checkpointing.md

```diff
@@ -1,34 +1,18 @@
----
-jupyter:
-  jupytext:
-    formats: ipynb,md
-    text_representation:
-      extension: .md
-      format_name: markdown
-      format_version: '1.3'
-      jupytext_version: 1.13.8
-  kernelspec:
-    display_name: Python 3 (ipykernel)
-    language: python
-    name: python3
----
-
```

Comment:
@cpgaffney1 we should keep the `jupytext` headers which as used for the `pre-commit` hooks. Please take a look at the [How to contribute](https://flax.readthedocs.io/en/latest/contributing.html) guide 🙂 .

Reply:
TLDR; run:
```
pip install pre-commit
pre-commit install
pre-commit run --all-files
```
Else your PR will fail if the notebooks and markdown files are not in sync.

Reply:
Thanks. I'm just going to make a new PR, merging is turning into more trouble than it's worth.